### ISSUE TITLE
Can not run with another TensorRT Engine in one program.

### ISSUE THREAD

**Issue by Water2style** [NVIDIA-AI-IOT/torch2trt#373]
I got a good model_trt by torch2trt, and it can run alone in a program.
But when i add an TensorRT engine in the program, it seems that program can not run an TensorRT engine and a model_trt together. But they can run correctly separately. 

If I want to run two models in one program, what should i do? Thanks!

**Comment by jaybdub** [comment]
Hi Water2style,

Thanks for reaching out!

Hmm.  I haven't seen this before personally.  Do you mind sharing the code you used to produce this issue?

Also, the following information may also be helpful

* PyTorch Version
* TensorRT Version
* Platform (ie: Jetson Xavier)


Best,
John

**Comment by Water2style** [comment]
> Hi Water2style,
> 
> Thanks for reaching out!
> 
> Hmm. I haven't seen this before personally. Do you mind sharing the code you used to produce this issue?
> 
> Also, the following information may also be helpful
> 
> * PyTorch Version
> * TensorRT Version
> * Platform (ie: Jetson Xavier)
> 
> Best,
> John

Sorry i cannot share the code,  but it just a strange try(tensoRT engine + trt.pth) , ocuurs : Cudnn Error configure:7 (CUDNN_STATUS_MAPPING_ERROR)  and CUDNN_STATUS_EXECUTION_FAILED
maybe i will give up this strange combination.  Thank you all the same John

**Comment by BarryKCL** [comment]
I encountered the same problem.when I use torch2trt and another TensorRT engine, an illegal memory access was encountered,but they can run correctly separately。

**Comment by xuatpham** [comment]
@BarryKCL  excuse me, have you solved it? I'm facing the same problem when running multi tensorRT_engines in one program. 
Thanks!

**Comment by fabiozappo** [comment]
Have anyone solved it? I'm facing the same problem when running a tensorRT_engine (yolov5) and another classification network (resnet_trt.pth) in the same program.

Thanks!

**Comment by xuatpham** [comment]
@fabiozappo  yes, I was able to solve it bro. All you need to do is just a simple trick:
initialize a tensor before loading the model. Good luck!

**Comment by fabiozappo** [comment]
> @fabiozappo yes, I was able to solve it bro. All you need to do is just a simple trick:
> initialize a tensor before loading the model. Good luck!

it works!
